modules/sw360/src/test/java/org/eclipse/sw360/antenna/sw360/rest/SW360ProjectClientTest.java

```diff
@@ -227,43 +226,12 @@ public void testCreateProjectWithSuccess() {
         assertEquals(PROJECT_EMAIL_VALUE_1, project.get_Embedded().getCreatedBy().getEmail());
     }
 
-    @Test( expected = HttpClientErrorException.class )
+    @Test
```

Comment:
I wouldn't get rid of these tests. The thing is, that in the success case, the returned object is a different instance right? You could test that the returned object is really the same instance (== instead of equals) and in the success case the object is only semantically equal.

Reply:
I agree with the `testCreateProjectWithBadStatusCode` test. 
the other ones I deleted, because the function they test is not existing anymore

Reply:
testProjectWithSuccess to test a successful run through the functionality?

Reply:
As discussed, I understand the issue now, and my comment is obsolete